docs/troubleshoot/troubleshoot-apiml.md

```diff
@@ -308,3 +308,29 @@ The Zowe started task needs to run under the same user ID as z/OSMF (typically I
 The hostname that is displayed in the details of the exception is a valid hostname. You can validate that the hostname is valid by using `ping` command on the same mainframe system. For example, `ping USILCA32.lvn.broadcom.net`. If it is valid, then the problem can be caused by insufficient privileges of your started task that is not allowed to do network access.
 
 You can fix it by setting up the security environment as described in the [Zowe documentation](../user-guide/configure-zos-system.html#configure-security-environment-switching).
+
+### Certificate error when using an external CA to deploy Zowe
```

Comment:
How about `Certificate error when using both an external certificate and Single Sign-On to deploy Zowe`?